pihole

```diff
@@ -52,6 +52,14 @@ function updateDashboardFunc {
     exit 1
 }
 
+function updateMainFunc {
+    echo "::: Fetching latest changes from Github..."
+    cd /etc/.pihole
+    ${SUDO} git pull
+    ${SUDO} /etc/.pihole/automated install/basic-install.sh
```

Comment:
Ah, OK. I didn't actually test that before I committed Just needs to be wrapped in quotes?